### ISSUE TITLE
Cannot install opensoundscape on Apple Silicon M1 chip

### ISSUE THREAD

**Issue by sammlapp** [kitzeslab/opensoundscape#533]
In a conda environment with python 3.8.13, grpcio fails to install for me. (However, other M1 chip users have not had issues. )

```
distutils.errors.CompileError: command 'gcc' failed with exit status 1
```

Installing the grpcio package with `conda install grpcio` worked and allowed me to `pip install opensoundscape`. However, I still had issues with libsndfile not being present, see #532

**Comment by sammlapp** [comment]
I can reliably get a working enviornment with opensoundscape installed on my M1 chip now by:
- creating a conda environment with python 3.8 or 3.9, eg `conda create -n opso python=3.8`
- `conda install grpcio`
- `pip install opensoundscape`
- Then follow [this workaround](https://github.com/ohmtech-rdi/eurorack-blocks/issues/444#issuecomment-1153155080) for the soundfile bug. I had to copy both the file `libsndfile.1.dylib` and `libsndfile.dylib` to the expected location rather than just one file as shown in that workaround. 

Not pretty, but it works for now.

**Comment by sammlapp** [comment]
With updated dependencies in #539, it seems that most of the M1 chip installation issues can be avoided by using a Python 3.9 environment. However, @syunkova still has trouble with building a pandas wheel.

**Comment by lydiakatsis** [comment]
Question related to this - If we install on M1 chip does OpSo utilise the GPU ?

**Comment by sammlapp** [comment]
> Question related to this - If we install on M1 chip does OpSo utilise the GPU ?

If you have Pytorch >=1.12 and MacOS >= 12.3, you can use the Apple Silicaon GPU by choosing the device name 'mps' (as I learned from [this](https://www.mrdbourke.com/pytorch-apple-silicon/) article). So in OpenSoundscape, if you have an object `mycnn` of class opensoundscape.torch.models.cnn.CNN, you can simply write `mycnn.device='mps'` to use the Apple Silicon GPU.

**Comment by sammlapp** [comment]
As far as I know, the recently released OpenSoundscape version 0.7.1 with Python 3.9 does not have installation errors on Apple Silicon chips, so I'm closing this issue for now.
